bundler/spec/install/gemfile/sources_spec.rb

```diff
@@ -379,14 +396,11 @@
                   rack
 
             PLATFORMS
-              #{lockfile_platforms}
+              #{lockfile_platforms("ruby")}
```

Comment:
Locally I had to add ruby to get this passing, now it's failing on windows.

Reply:
In most situations `lockfile_platforms` should include `ruby` by default since #5700. What's the error if you revert the change? I can't reproduce it locally.